src/store/models/network.ts

```diff
@@ -53,6 +55,32 @@ export interface AutoMinerModel {
   mining: boolean;
 }
 
+interface Paths {
+  tlsCert: string;
+  adminMacaroon: string;
+  invoiceMacaroon: string;
+  readonlyMacaroon: string;
+}
+
+interface Node {
```

Comment:
Why create a duplicate node type here when we have all of the node types in [types.ts](https://github.com/jamaljsr/polar/blob/master/src/shared/types.ts#L9). You can just accept `CommonNode` as the parameter then determine the correct node type based on the `implementation` field.